### PR TITLE
Create separate OndaEDFSchemas sub-package

### PR DESCRIPTION
**Pull request by kleinschmidt**
This partially addresses #50 by creating OndaEDFSchemas.jl inside this repo and adding tests, CI, and codecove for it.  We will need to wait to move OndaEDF.jl over to use this until it is registered though to avoid teh ArrowTypes situation.

### CODE REVIEW

File: codecov.yml [hunk 1]
```diff
@@ -1 +1,10 @@
+flag_management:
+  default_rules: # the rules that will be followed for any flag added, generally
+    carryforward: true
+    statuses:
+      - type: project
+        target: auto
+        threshold: 1%
+      - type: patch
+        target: 90%
 comment: off
```
Comment (palday) [inline codecov.yml:10]: do we need this or can we just take advantage of org settings?

**Review by palday — APPROVED** [review]

File: codecov.yml (hunk 1 above)
Comment (kleinschmidt) [reply codecov.yml:10]: we need at least the `flag_management` thing, let me try without that...

Comment (kleinschmidt) [reply codecov.yml]: null value not allowed there but I can ditch all teh statuses: bits

**Comment by kleinschmidt** [comment]
Gonna merge this and worry about getting CI perfect later :)

**Pull request merged**